src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        }
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+    }
+  }
```

Comment:
Where is default case?
Please check [checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md)
![2023-08-04_15h53_52](https://github.com/mate-academy/js_stateful-object/assets/61904995/33cc92cc-f25f-4b64-9075-cf2120c27c8e)